src/TheOpenNetwork/Message.h

```diff
@@ -0,0 +1,21 @@
+// Copyright © 2017-2022 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+#include "Everscale/Messages.h"
+
+using namespace TW::Everscale;
+
+namespace TW::TheOpenNetwork {
+
+class Message : public Everscale::Message {
```

Comment:
This is literally the same Message object, which was declared in early [TL-B schemas](https://github.com/ton-blockchain/ton/blame/master/crypto/block/block.tlb#L147) and hasn't been changed for years in both blockchains, because it's a part of the main core. The difference is only in a serialization approach `intoCell` (we use cell references instead of writing raw cells). `setBody` and `setStateInit` are being reused w/o changes, class fields are the same. 

Could you please explain why you think it would be better to separate and duplicate class definitions here? We reuse many components of that schema "as is". E.g. message headers, info and more.